conn/physic/units.go

```diff
@@ -217,17 +222,15 @@ func (f *Frequency) Set(s string) error {
 		if e, ok := err.(*parseError); ok {
 			switch e.err {
 			case errOverflowsInt64:
-				found, _ := containsUnitString(s[n:], "Hertz", "Hz")
-				return &parseError{
-					msg: "maximum value for \"" + found + "\" is " + e.msg + "n" + found,
-					err: errors.New(""),
-				}
+				return errors.New("maximum value is " + maxFrequency.String())
 			case errUnderflowsInt64:
```

Comment:
FTR, underflow is having a number too small, i.e. 1pHz. A negative number is an overflow but in the negative side.

https://en.wikipedia.org/wiki/Arithmetic_underflow